src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -321,13 +320,10 @@ function ReportActionCompose({
             return;
         }
 
-        const viewTag = animatedRef();
-        const viewName = 'RCTMultilineTextInputView';
-        const updates = {text: ''};
         // We are setting the isCommentEmpty flag to true so the status of it will be in sync of the native text input state
         runOnJS(setIsCommentEmpty)(true);
         runOnJS(resetFullComposerSize)();
-        updatePropsPaperWorklet(viewTag, viewName, updates); // clears native text input on the UI thread
+        setNativeProps(animatedRef, {text: ''}); // clears native text input on the UI thread
```

Comment:
`setNativeProps` for `react-native` is [deprecated](https://github.com/react-navigation/react-navigation/issues/11137). 

To migrate to Fabric architecture, we have to remove all usages of `setNativeProps`.

Could we confirm if it's OK to use `setNativeProps` from `react-native-reanimated`?

Reply:
`setNativeProps` from `react-native-reanimated` is something different than `setNativeProps` from `react-native`; it works on the UI thread and it doesn't have some of the problems of `setNativeProps` from `react-native`, see docs here: https://docs.swmansion.com/react-native-reanimated/docs/advanced/setNativeProps/

It's used here as a workaround to fix an underlying issue in RN's TextInput component until we have a better solution.

Finally, `setNativeProps` from `react-native-reanimated` in fact a public wrapper on `updatePropsPaperWorklet` (which is meant to be used only internally) so this PR does not introduce any functional changes.

Reply:
Does this answer your question? @rushatgabhane 😄

Reply:
yes thank you so much, Tomek!!